src/main/kotlin/com/faforever/userservice/config/SecurityConfiguration.kt

```diff
@@ -27,6 +27,7 @@ class SecurityConfiguration {
                 authorize("/robots.txt", permitAll)
                 authorize("/favicon.ico", permitAll)
                 authorize("/login", permitAll)
+                authorize("/banned", permitAll)
                 authorize("/consent", permitAll)
```

Comment:
I'm wondering if we get too many top level routes. Maybe we should move /banned and consent under /login ? Opinions?

It's now or never :stuck_out_tongue:

Reply:
I was thinking the same thing

Reply:
Maybe we keep consent as top level though what do you think?

Reply:
otherwise just need to change hydra yaml